pkg/asset/installconfig/clusterid.go

```diff
@@ -19,7 +19,7 @@ func (a *clusterID) Dependencies() []asset.Asset {
 func (a *clusterID) Generate(map[asset.Asset]*asset.State) (*asset.State, error) {
 	return &asset.State{
 		Contents: []asset.Content{
-			{Data: []byte(uuid.NewUUID().String())},
+			{Data: []byte(uuid.NewUUID())},
```

Comment:
Previous discussion starting [here][1].  I'd still prefer to stick to pborman/uuid and just use the random (version 4) UUID via  [`[]byte(uuid.New())`][2].

[1]: https://github.com/openshift/installer/pull/120#discussion_r212722175
[2]: https://godoc.org/github.com/pborman/uuid#New